cluster/saltbase/salt/calico/master.sls

```diff
@@ -0,0 +1,70 @@
+{% if pillar.get('policy_provider', '').lower() == 'calico' %}
+
+calicoctl:
+  file.managed:
+    - name: /usr/bin/calicoctl
+    - source: https://github.com/projectcalico/calico-docker/releases/download/v0.18.0/calicoctl
+    - source_hash: sha256=694d2e0d0079980d75bc807fcb6626b18e5994638aa62743d45b906e742a0eed
+    - makedirs: True
+    - mode: 744
+
+calico-policy:
```

Comment:
We shouldn't be installing this.  It's going to go away once the real NetworkPolicy objects are introduced.